jdk17-image/image.yaml

```diff
@@ -0,0 +1,160 @@
+schema_version: 1
+
+name: "jboss-eap-7-tech-preview/eap74-openjdk17-openshift-rhel8"
+description: "Red Hat JBoss Enterprise Application Platform 7.4 OpenShift container image."
+version: "7.4.5"
+from: "registry.redhat.io/ubi8/ubi:latest"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-openjdk17-builder-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for building and running JavaEE applications on JBoss EAP 7.4"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP 7.4"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "builder,javaee,eap,eap7"
+    - name: "io.openshift.s2i.scripts-url"
+      value: "image:///usr/local/s2i"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+      - name: DISABLE_LEGACY_SECURITY
+        value: "true"
+      - name: CONFIGURE_ELYTRON_SSL
+        value: "true"
+      - name: SSO_FORCE_LEGACY_SECURITY
+        value: "false"
+      - name: HTTPS_ENABLE_HTTP2
+        value: "true"
+      - name: SCRIPT_DEBUG
+        description: If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed.
+        example: "true"
+      - name: AB_PROMETHEUS_JMX_EXPORTER_PORT
+        value: 9799
+ports:
+    - value: 8443
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: 0.45.4
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
+                  ref: EAP_745_CR1_JDK11
```

Comment:
I don't see this tag in  https://github.com/jboss-container-images/jboss-eap-modules/tags. There is only a `EAP_745_CR1`. Besides, is it normal to use a `_JDK11` tag to build the JDK17 images?

Reply:
This is merged and tagged now.